environment.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
     - toytree
     - toyplot
     - pytest
+    - zarr<3
```

Comment:
This is safer indeed. We'll probably need some work in xarray-simlab to support Zarr v3.

Reply:
Thanks for taking a look! I'll merge it on then.